desk/app/chat.hoon

```diff
@@ -1754,12 +1762,17 @@
     |=  j=join:c
     ^+  ca-core
     ?>  |(=(p.group.j src.bowl) =(src.bowl our.bowl))
-    =.  chats  (~(put by chats) chan.j *chat:c)
+    =|  =chat:c
+    =.  net.chat
+      ?:  =(our.bowl p.chan.j)  [%pub ~]
+      [%sub p.chan.j | %chi ~]
+    =.  chats  (~(put by chats) chan.j chat)
     =.  ca-core  (ca-abed chan.j)
     =.  last-read.remark.chat  now.bowl
     =.  group.perm.chat  group.j
     =.  cor  (give-brief flag/flag ca-brief)
-    ca-sub
```

Comment:
we don't need sub here because eventually we do `ca-make-chi` and that subs?

Reply:
Correct.